db/citations.yml

```diff
@@ -10135,6 +10135,37 @@ thamp;prb13:
   volume: '88'
   wwwemail: ''
   year: '2013'
+thatc;arxiv21:
+  author:
+    - Zachary Thatcher
+    - Chia-Hao Liu
+    - Long Yang
+    - Brennan C. McBride
+    - Gia Thinh Tran
+    - Allison Wustrow
+    - Martin A. Karlsen
+    - James R. Neilson
+    - Dorthe B. Ravnsbæk
+    - Simon J. L. Billinge
+  doi: ''
+  entrytype: article
+  facility: nslsii, xpd, desy, P02.1
+  grant: efrc18
+  journal: arXiv:2112.13300 [cond-mat]
```

Comment:
this should be in `note`.  for the journal just put `arxiv`

Reply:
resolved